pyzeebe/job/job.py

```diff
@@ -58,17 +58,19 @@ def set_failure_status(self, message: str) -> None:
 
         """
         if self.zeebe_adapter:
-            self.zeebe_adapter.fail_job(job_key=self.key, message=message)
+            self.zeebe_adapter.fail_job(job_key=self.key, retries=self.retries, message=message)
         else:
             raise NoZeebeAdapterError()
 
-    def set_error_status(self, message: str) -> None:
+    def set_error_status(self, message: str, error_code: str = None) -> None:
```

Comment:
Shouldn't `""` be the default value? How does Zeebe handle a when it receives `null`?

Reply:
You're right, I thought that `None` would reproduce the previous behaviour of not sending anything, but after I tested it, `None` just get stringified and received by Zeebe as `"None"`. I'm changing it to `""`